processor/src/main/java/org/mapstruct/ap/internal/util/accessor/ExecutableElementAccessor.java

```diff
@@ -28,4 +28,9 @@ public TypeMirror getAccessedType() {
     public ExecutableElement getExecutable() {
         return element;
     }
+
+    @Override
+    public String toString() {
+        return element.getSimpleName().toString() + "( . )";
```

Comment:
Why not just use the `element.toString()` instead? If we want to use something specific in the printing of the messager we should pass that in the arguments, and not do it in the `toString`. For debugging it would be much easier to just have `element.toString()` here.